### PR TITLE
Remove istanbul hacks

### PR DESCRIPTION
**Pull request by existentialism**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | 
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | Yes
| Documentation PR         | 
| Any Dependency Changes?  |
| License                  | MIT

Should no longer be needed after https://github.com/istanbuljs/istanbuljs/pull/135

### CODE REVIEW

**Comment by nicolo-ribaudo** [comment]
Can you also update the version in package.json?

File: .babelrc.js [hunk 1]
```diff
@@ -40,31 +10,32 @@ const config = {
...
+        "babel-plugin-transform-charcodes",
+        ["@babel/transform-for-of", { assumeArray: true }],
+      ],
+    },
+  ],
 };
 
 if (process.env.BABEL_ENV === "cov") {
   config.auxiliaryCommentBefore = "istanbul ignore next";
-  config.plugins.push(istanbulHacks);
```
Comment (nicolo-ribaudo) [inline .babelrc.js]: Probably this should push `babel-plugin-istanbul`, since `istanbulHacks` inherited from it?

Comment (existentialism) [reply .babelrc.js]: Fixed!

**Comment by existentialism** [comment]
@nicolo-ribaudo most of the deps that are bumped are transient via `babel-plugin-istanbul`. I went ahead and bumped to the latest patch anyway (which was published Aug 2017).

**Pull request merged**